packages/office-addin-cli/src/cli.ts

```diff
@@ -18,6 +19,15 @@ commander.on("command:*", function () {
   commander.help();
 });
 
+commander
+  .command("convert")
+  .option("-m, --manifest <manifest-path>", "Specify the location of the manifest file.  Default is './manifest.xml'")
+  .option(
+    "-p, --packageJson <packageJson-path>",
+    "Specify the location of the package.json file.  Default is './package.json'"
```

Comment:
Doesn't this always have to been in the root?  Do we need to make this an option?